packages/libraries/core/src/components/quantity/ods-quantity-controller.ts

```diff
@@ -33,13 +33,15 @@ export class OdsQuantityController extends OdsComponentController<OdsQuantity> {
   }
 
   private onBlur() {
```

Comment:
could you update the unit test that is testing the blur of the component with a value of 0 for min and/or max ?
and check that value is ok

Reply:
There is no test for onBlur since it is private